modules/nw-route-specific-annotations.adoc

```diff
@@ -5,7 +5,7 @@
 [id="nw-route-specific-annotations_{context}"]
 = Route-specific annotations
 
-The Ingress Controller can set the default options for all the routes it exposes. An individual route can override some of these defaults by providing specific configurations in its annotations.
+The Ingress Controller can set the default options for all the routes it exposes. An individual route can override some of these defaults by providing specific configurations in its annotations. Route annotations for operator-managed routes are unsupported.
 
```

Comment:
I might have gone with "Red Hat does not support adding a route annotation to an operator-managed route." but I'm making an assumption that I understand the material.

It's OK with or without the revision because the context is high--the table that follows the sentence catalogs the annotations. (Maybe some day, we should change the column name from Variable to Annotation.)

Reply:
How does a user determine an 'operator-managed route'? 

Being able to show a users what routes they can/shouldn't touch it important! 
We should also file an insights nomination for things like this! https://access.redhat.com/labs/proactiveissuestracker/ 
In short if insights can be used to spot when an annotation is set on an 'operator-managed' route; then remediation steps to users can be provided; in the console or OCM on how to take corrective action. 

Thus further enforcing or highlighting this critical call out. 

cc: @jan-zmeskal for guidance and help in articulating a rule/ensuring we collect the right data to file/progress such a rule.

Reply:
> How does a user determine an 'operator-managed route'?

@sferich888: Do you think adding a sentence directing users to to the [OperatorHub](https://docs.openshift.com/container-platform/4.7/operators/understanding/olm-understanding-operatorhub.html) topic give enough supporting context until larger issues can be addressed?